README: Correct the instructions for deployment (#58)
ARMmbed/mbed-os-example-thread-statistics: README.md

You can build this project with all supported [Mbed OS build tools](https://os.mbed.com/docs/mbed-os/latest/tools/index.html). However, this example project specifically refers to the command-line interface tool [Arm Mbed CLI](https://github.com/ARMmbed/mbed-cli#installing-mbed-cli):

1. [Install Mbed CLI](https://os.mbed.com/docs/mbed-os/latest/quick-start/offline-with-mbed-cli.html).

1. Clone this repository on your system, and change the current directory to where the project was cloned and deploy:

    ```bash
    $ git clone git@github.com:ARMmbed/mbed-os-example-thread-statistics.git && cd mbed-os-example-thread-statistics
    $ mbed deploy
    ```

    Alternatively, you can download the example project with Arm Mbed CLI using the `import` subcommand:

    ```bash
    $ mbed import mbed-os-example-thread-statistics && cd mbed-os-example-thread-statistics

>     ```

Reviewer: Just show the import command. No need to have two different sets of instructions.

Author: @evedon Done. Please review

You can build this project with all supported [Mbed OS build tools](https://os.mbed.com/docs/mbed-os/latest/tools/index.html). However, this example project specifically refers to the command-line interface tool [Arm Mbed CLI](https://github.com/ARMmbed/mbed-cli#installing-mbed-cli):

1. Install Mbed CLI.

> 1. Clone this repository and deploy the Mbed OS: `mbed import mbed-os-example-thread-statistics`

Reviewer: ```suggestion
1. From the command-line, import the example: `mbed import mbed-os-example-thread-statistics`
```
